app/views/user_mailer/creatorship_notification.html.erb

```diff
@@ -14,7 +16,6 @@
   </p>
 
   <p>
-    <%= t(".html.remove",
-    edit: style_link(t(".html.edit", type: creation_type), edit_polymorphic_url(@creation))).html_safe %>
+    <%= t(".html.remove_#{creation_type}", edit: style_link(t(".html.edit_#{creation_type}"), edit_polymorphic_url(@creation))).html_safe %>
```

Comment:
Following some conversation about links with Translation, what we want to do is make the name of the link placeholder they see in Phrase (currently `edit`) is equal to the key for the link text plus the suffix `_link`. By establishing a pattern, we save them a bit of time in preparing their documents for translation.

So could you change `edit:` here to `"edit_#{creation_type}_link":`?

It's a little ugly, but I checked and the tests pass.

Reply:
Got it, yes